R/account-activity.R

```diff
@@ -7,7 +7,7 @@
 get_account_activity <- function(account_number, start_date, end_date){
   url <- paste0(
     "https://api.apexclearing.com/activities-provider/api/v1/activities/",
-    taxable,
+    account_number,
```

Comment:
This is a straight up bug I fixed - I had a test script where I defined `taxable <- "my_num"` and was doing some dev work off of that. This should be the account number from the function parameters.